Makefile

```diff
@@ -1,4 +1,4 @@
-CPPFLAGS += -I llvm-backend/build/include -I llvm-backend/include -I vm-c -I dummy-version -I plugin -I vm-c/kevm -I plugin-c -I libff/build/install/include
+CPPFLAGS += -I llvm-backend/build/include -I llvm-backend/include -I vm-c -I dummy-version -I plugin -I vm-c/kevm -I plugin-c -I libff/build/install/include -L /home/anvacaru/rv/proxygen/proxygen/_build/lib/libproxygenhttpserver.a -L /home/anvacaru/rv/proxygen/proxygen/_build/lib/libproxygen.a -I /home/anvacaru/rv/proxygen/proxygen/_build/deps/include/
```

Comment:
`-L` is a search path, not where the library itself is. Also, we shouldn't hardcode absolute paths.
Assuming the submodule is at `deps/proxygen`
So for example, `-L /home/anvacaru/rv/proxygen/proxygen/_build/lib/libproxygenhttpserver.a` should become `-L deps/proxygen/_build/lib`, and you link against the actual file with `-l proxygenhttpserver`